package.json

```diff
@@ -72,7 +72,7 @@
     "@mux/mux-node": "^8.5.1",
     "@mux/mux-player-react": "^2.6.0",
     "@oaknational/oak-components": "^0.38.0",
-    "@oaknational/oak-curriculum-schema": "^1.9.0",
+    "@oaknational/oak-curriculum-schema": "^1.14.0",
```

Comment:
if you're updating the schema will need to add the pupil page fixes

Reply:
Just doing your review atm, so will wait for it to go in first